modules/influxdb/examples_test.go

```diff
@@ -41,3 +43,69 @@ func ExampleRun() {
 	// Output:
 	// true
 }
+
+func ExampleRun_v2() {
+	// runInfluxV2Container {
+	ctx := context.Background()
+
+	username := "username"
+	password := "password"
+	org := "org"
+	bucket := "bucket"
+	authEnabled := true
+	token := "influxdbv2token"
+
+	influxdbContainer, err := influxdb.Run(ctx,
+		"influxdb:2.7.11",
+		influxdb.WithV2Env(influxdb.InfluxDBV2Config{
+			Username:    &username,
+			Password:    &password,
+			Org:         org,
+			Bucket:      bucket,
+			Token:       &token,
+			AuthEnabled: &authEnabled,
+		}),
+	)
+	defer func() {
+		if err := testcontainers.TerminateContainer(influxdbContainer); err != nil {
+			log.Printf("failed to terminate container: %s", err)
+		}
+	}()
+	if err != nil {
+		log.Printf("failed to start container: %s", err)
+		return
+	}
+	// }
+
+	state, err := influxdbContainer.State(ctx)
+	if err != nil {
+		log.Printf("failed to get container state: %s", err)
+		return
+	}
+
+	fmt.Println(state.Running)
+
+	// Query the InfluxDB API to verify the setup
+	url, err := influxdbContainer.ConnectionUrl(ctx)
+	if err != nil {
+		log.Printf("failed to get host: %s", err)
+		return
+	}
+
+	// Initialize a new InfluxDB client
+	client := influxdb2.NewClientWithOptions(url, token, influxdb2.DefaultOptions())
+	defer client.Close()
+
+	// Get the bucket
+	influxBucket, err := client.BucketsAPI().FindBucketByName(ctx, bucket)
+	if err != nil {
+		log.Printf("failed to get bucket: %s", err)
+		return
+	}
+
+	log.Printf("Got bucket: %s", influxBucket.Name)
```

Comment:
bug: this is not honouring the testable example format, causing the tests to fail https://github.com/testcontainers/testcontainers-go/actions/runs/14207921652/job/39809564404#step:7:437
```suggestion
	fmt.Println(influxBucket.Name)
```